packages/fsml/fsml.0.2.1/opam

```diff
@@ -0,0 +1,31 @@
+opam-version: "2.0"
+synopsis:
+  "A library for describing and describing synchronous finite state machines"
+maintainer: ["Jocelyn Sérot <jocelyn.serot@uca.fr>"]
+authors: ["Jocelyn Sérot <jocelyn.serot@uca.fr>"]
+license: "MIT"
+homepage: "http://github.com/jserot/fsml"
+doc: "http://jserot.github.io/fsml"
+bug-reports: "jocelyn.serot@uca.fr"
+depends: [
+  "dune" {>= "2.6"}
+  "ppxlib" {>= "0.13.0"}
+  "ppx_deriving_yojson" {>= "3.5.3"}
+  "menhir" {>= "20200624"}
+  "ocaml" {>= "4.10"}
+]
+build: [
+  ["dune" "subst"] {pinned}
+  [
+    "dune"
+    "build"
+    "-p"
+    name
+    "-j"
+    jobs
+    "@install"
+    "@runtest" {with-test}
+    "@doc" {with-doc}
+  ]
+]
+dev-repo: "git+https://github.com/jserot/fsml.git"
```

Comment:
It is missing the url section pointing at the sources here
```suggestion
dev-repo: "git+https://github.com/jserot/fsml.git"
url {
  src: "https://github.com/jserot/fsml/archive/0.2.1.tar.gz"
  checksum: "sha512=d649da27e4f08caa043736523c62a357df951c79a5589cd1e2d5c264540d0da6088a2b24bb51401cb54f6be79444bcdac86bccb496463deb75b4646b35cd7ba2"
}
```
Maybe you need to update opam-publish, but sometimes it is just not picking it up -- no idea why.

Reply:
Anyways, this should do, no action needed.